consensus/enclave/build.rs

```diff
@@ -2,15 +2,37 @@
 
 //! Build script for consensus service daemon
 
-use cargo_emit::{rustc_link_lib, rustc_link_search};
+use cargo_emit::rustc_cfg;
 use mc_util_build_script::Environment;
-use mc_util_build_sgx::{Edger8r, SgxEnvironment, SgxMode};
+use mc_util_build_sgx::{Edger8r, SgxEnvironment, SgxLibraryCollection, SgxMode};
+use pkg_config::{Config, Error as PkgConfigError, Library};
+
+const SGX_LIBS: &[&str] = &["libsgx_urts", "libsgx_epid"];
+const SGX_SIMULATION_LIBS: &[&str] = &["libsgx_urts_sim", "libsgx_epid_sim"];
+const SGX_VERSION: &str = "2.9.101.2";
```

Comment:
`// Changing this version is a breaking change, you must update the crate version if you do.` like in other places?

Reply:
Yeah, I'll update, though ideally this kind of stuff can be exported by the dependencies, similar to the other EDL crates, which should isolate the untrusted enclave from needing to deal with pkg-config detection directly.